src/Subscriber/AutoLabelFromContentSubscriber.php

```diff
@@ -43,16 +39,16 @@ public function onPullRequest(GitHubEvent $event)
         }
 
         // the PR body usually indicates if this is a Bug, Feature, BC Break or Deprecation
-        if (preg_match('/\|\s*Bug fix\?\s*\|\s*yes\s*/i', $prBody, $matches)) {
+        if (preg_match('/\|\s*Bug fix\?\s*\|\s*yes\s*/i', (string) $prBody, $matches)) {
             $prLabels[] = 'Bug';
         }
-        if (preg_match('/\|\s*New feature\?\s*\|\s*yes\s*/i', $prBody, $matches)) {
+        if (preg_match('/\|\s*New feature\?\s*\|\s*yes\s*/i', (string) $prBody, $matches)) {
```

Comment:
Those explicit string casts are not needed IMO.

Reply:
Will check and remove

Reply:
Done @stof